osquery/tables/system/windows/startup_items.cpp

```diff
@@ -103,16 +124,17 @@ QueryData genStartupItems(QueryContext& context) {
       }
     }
 
+    parseStartupPath(startup.at("data"), r);
+
     r["status"] = regex_match(startup.at("status"), kStartupDisabledRegex)
                       ? "disabled"
                       : "enabled";
     r["name"] = startup.at("name");
-    r["path"] = startup.at("data");
     r["source"] = startup.at("key");
     r["type"] = "Startup Item";
     results.push_back(r);
   }
   return results;
 }
-}
-}
+} // namespace tables
```

Comment:
@theopolis I've had this pop up on my commits as well, is this something we turned on with our clang formatting? Or was this a new feature in the newer versions? Are we cool with the namespace tagging? My two-cents is that this is pretty neat, and I think we should do it.